data_generation/fractal_graph_expansions/run.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
-output_prefix="16_32_movie_lens_expansion"
+input_file="/home/belletti/ml-20m/ratings.csv"
```

Comment:
Do you mean to commit these paths?

Reply:
Thanks, I commented a change to delete belletti from the path.